roles/uperf-bench/tasks/main.yml

```diff
@@ -47,8 +47,8 @@
             app : uperf-bench-server
         spec:
```

Comment:
Move these to templates. That way we can put the logic there. Use sysbench as an example.